src/language/branches.md

```diff
@@ -151,8 +159,8 @@ token = switch (charCode) {
 
 ### Exhaustiveness checking
 
-Exhaustiveness checking is a feature of switches that reports a compile-time
-error if it's possible for a value entering the switch to not match any of the cases.
+Exhaustiveness checking is a feature that reports a compile-time
```

Comment:
**nit:** Consider shortening this.
```suggestion
Exhaustiveness checking reports a compile-time
```

Reply:
Hmm.. I'm gonna keep it because "exhaustiveness checking" sounds like a verb, but it's kind of actually a "thing"... I guess. I think clarity helps here.